conformance_tests/sysman/test_sysman_fabric/src/test_sysman_fabric.cpp

```diff
@@ -289,6 +289,127 @@ TEST_F(
   }
 }
 
+TEST_F(
+    FABRICPORT_TEST,
+    GivenValidFabricPortHandleWhenRetrievingMultiPortThroughputsThenValidThroughputAreReturned) {
+  for (auto device : devices) {
+    uint32_t count = 0;
+    auto fabric_port_handles = lzt::get_fabric_port_handles(device, count);
+    if (count == 0) {
+      FAIL() << "No handles found: "
+             << _ze_result_t(ZE_RESULT_ERROR_UNSUPPORTED_FEATURE);
+    }
+
+    auto multiport_throughPuts = lzt::get_multiport_throughputs(
+        device, count, fabric_port_handles.data());
+    ASSERT_EQ(fabric_port_handles.size(), count);
+
+    for (auto fabric_port_throughput : multiport_throughPuts) {
+      EXPECT_LT(fabric_port_throughput.rxCounter, UINT64_MAX);
+      EXPECT_LT(fabric_port_throughput.txCounter, UINT64_MAX);
+      EXPECT_LT(fabric_port_throughput.timestamp, UINT64_MAX);
+    }
+  }
+}
+
+TEST_F(
+    FABRICPORT_TEST,
+    GivenValidFabricPortHandleWhenRetrievingMultiportThroughputsThenValuesAreSameAsGetThroughput) {
+  for (auto device : devices) {
+    uint32_t count = 0;
+    uint32_t i = 0;
+    auto fabric_port_handles = lzt::get_fabric_port_handles(device, count);
+    if (count == 0) {
+      FAIL() << "No handles found: "
+             << _ze_result_t(ZE_RESULT_ERROR_UNSUPPORTED_FEATURE);
+    }
+
+    // Taking values from zesFabricPortGetMultiPortThroughput
+    auto multiport_throughputs = lzt::get_multiport_throughputs(
+        device, count, fabric_port_handles.data());
+    ASSERT_EQ(fabric_port_handles.size(), count);
+
+    // Taking values from zesFabricPortGetThroughput and comparing it with
+    // multiport_throughPuts
+    for (auto fabric_port_handle : fabric_port_handles) {
+      ASSERT_NE(nullptr, fabric_port_handle);
+      auto throughput = lzt::get_fabric_port_throughput(fabric_port_handle);
+      EXPECT_EQ(throughput.rxCounter, multiport_throughputs[i].rxCounter);
+      EXPECT_EQ(throughput.txCounter, multiport_throughputs[i].txCounter);
+      EXPECT_EQ(throughput.timestamp, multiport_throughputs[i].timestamp);
+      i++;
+    }
+  }
+}
+
+TEST_F(
+    FABRICPORT_TEST,
+    GivenPortHandleVectorAsNullWhenRetrievingMultiportThroughputsThenRequiredErrorShouldBeReturned) {
```

Comment:
Please recheck the description of the test.
Something like, if fabricPortsCouldNotbeEnumeratedThenReturnUNSUPPORTED